tsl/src/continuous_aggs/create.c

```diff
@@ -1071,7 +1071,7 @@ is_valid_bucketing_function(Oid funcid)
 {
 	FuncInfo *finfo = ts_func_cache_get_bucketing_func(funcid);
 
-	return finfo != NULL && finfo->is_timescaledb_func && finfo->nargs == 2;
+	return finfo != NULL && (finfo->origin != ORIGIN_POSTGRES) && finfo->nargs == 2;
```

Comment:
This affects the public cagg api.

Reply:
I don't see how exactly. Maybe you could give an example?

Reply:
I mean: it changes the implementation for sure. I don't see how the API is affected.

Reply:
Maybe check `ORIGIN_TIMESCALE` || `ORIGIN_TIMESCALE_EXPERIMENTAL` in such case? Feel like this change allows any function, not only provided by the extension, if I understood correctly :)

Reply:
@pmwkaa Thanks for the feedback. Fixed.